src/toolchain.rs

```diff
@@ -333,8 +333,12 @@ impl<'a> Toolchain<'a> {
         // Should push the cargo fallback into a custom toolchain type? And then
         // perhaps a trait that create command layers on?
         if let Some(cmd) = self.maybe_do_cargo_fallback(binary)? {
+            info!("`cargo` is unavailable for the active toolchain");
+            info!("falling back on {:?}", cmd.get_program());
```

Comment:
Suggest that "falling back" usually pairs with "to" rather than "on"?

Reply:
@djc I started with `falling back to` and my spell checker is not happy with it: https://en.wiktionary.org/wiki/fall_back_on

Reply:
Reverted to `falling back to` since it looks more popular across our org.